app/src/main/java/com/gh4a/adapter/timeline/EventViewHolder.java

```diff
@@ -258,69 +258,96 @@ private CharSequence formatEvent(final IssueEvent event, final User user, int ty
         if (textBase == null) {
             textBase = mContext.getString(textResId, getUserLoginWithBotSuffix(user));
         }
+
         SpannableStringBuilder text = StringUtils.applyBoldTags(textBase, typefaceValue);
+        replaceCommitPlaceholder(event, text);
+        replaceLabelPlaceholder(event, text);
+        replaceBotPlaceholder(text);
+        replaceSourcePlaceholder(event, text);
+        replaceTimePlaceholder(event, text);
+        return text;
+    }
 
+    private void replaceCommitPlaceholder(IssueEvent event, SpannableStringBuilder text) {
         int pos = text.toString().indexOf("[commit]");
         if (event.commitId() != null && pos >= 0) {
```

Comment:
Please (at least ;-) ) invert the logic here and return early if preconditions aren't met, so indention depth is reduced now we don't need the block for variable scoping anymore:

```
if (pos < 0 || event.commitId() == null) {
    return;
}
```

(same below)

Reply:
Here's another benefit of extracting methods, the ability to return early :wink: 
Will do it in a moment